src/main/java/herrekt/Storage.java

```diff
@@ -0,0 +1,81 @@
+package herrekt;
+
+import herrekt.taskmanager.Deadline;
+import herrekt.taskmanager.Event;
+import herrekt.taskmanager.Task;
+import herrekt.taskmanager.Todo;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+import static java.util.stream.Collectors.toList;
+
+public class Storage {
+
+    private final String filePath;
+
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    public List<Task> load() throws FileNotFoundException {
+        List<String> stringList = loadSaveAsStringList(filePath);
+        return convertStringListToTaskList(stringList);
```

Comment:
It would help to [name](https://nus-cs2113-ay2122s1.github.io/website/se-book-adapted/chapters/codeQuality.html#use-nouns-for-things-and-verbs-for-actions) what list to be returned, instead of implementing a method call as the `return` statement.